src/components/players/detailsPlaceholder.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+
+import styles from './detailsPlaceholder.module.scss';
+
+const DetailsPlaceholder = () => {
+    return (
+        <div className={styles.playerDetailsContainer}>
+            <div className={styles.picture}/>
+            <div className={styles.underline}/>
+            <div className={styles.box}>
+                <div className={styles.textBox}>
+                    <div className={styles.text1}/>
+                    <div className={styles.text2}/> 
+                    <div className={styles.text3}/>
```

Comment:
It's not convention to use numbers as part of style variables or variables in general

```
                    <div className={styles.title}/>
                    <div className={styles.subtitle}/> 
                    <div className={styles.details}/>
```

or

```
                    <div className={styles.textA}/>
                    <div className={styles.textB}/> 
                    <div className={styles.textC}/>
```

or

```
                    <div className={styles.textOne}/>
                    <div className={styles.textTwo}/> 
                    <div className={styles.textThree}/>
```